src/STEMMUS_SCOPE.m

```diff
@@ -616,17 +636,21 @@
                 RS = (EXCESS - DSTOR) / Delt_t;
             else
                 AVAIL = AVAIL0 - Evap(KT);
-                EXCESS = (AVAIL + HBoundaryFlux.QMT) * Delt_t;
+                EXCESS = (AVAIL + HBoundaryFlux.QMT) * Delt_t; % (unit converstion from cm/sec to cm/30mins)
                 if abs(EXCESS / Delt_t) <= 1e-10
                     EXCESS = 0;
                 end
-                DSTOR = min(EXCESS, DSTMAX);
-                RS(KT) = (EXCESS - DSTOR) / Delt_t;
+                DSTOR = min(EXCESS, DSTMAX); % Depth of depression storage at end of current time step
+                % Next line is commented and Surface runoff is re-calculated using different approach (the following 3 lines)
+                % RS(KT) = (EXCESS - DSTOR) / Delt_t; % surface runoff, (unit conversion from cm/30mins to cm/sec)
+                R_Dunn(KT) = ForcingData.R_Dunn(KT); % (cm/sec)
+                R_Hort(KT) = ForcingData.R_Hort(KT); % (cm/sec)
+                RS(KT) = R_Hort(KT) + R_Dunn(KT); % total surface runoff (cm/sec)
```

Comment:
```suggestion
                RS(KT) = ForcingData.R_Hort(KT) + ForcingData.R_Dunn(KT); % total surface runoff (cm/sec)
```

Reply:
no need to define variables `R_Hort` and `R_Dunn` here as they are already in the structure `ForcingData`